packages/react-components/react-charts-preview/src/utilities/utilities.ts

```diff
@@ -221,9 +237,11 @@ export function createDateXAxis(
   options?: Intl.DateTimeFormatOptions,
   timeFormatLocale?: d3TimeLocaleDefinition,
   customDateTimeFormatter?: (dateTime: Date) => string,
+  useUTC?: string| boolean,
 ) {
   const { domainNRangeValues, xAxisElement, tickPadding = 6, xAxistickSize = 6, xAxisCount = 6 } = xAxisParams;
-  const xAxisScale = d3ScaleUtc()
+  const xAxisScale = useUTC ? d3ScaleUtc() : d3ScaleTime();
+  xAxisScale
```

Comment:
you need to assign the chained result to some variable

Reply:
Did not get

Reply:
We are assigning to xAxisScale and then using it to calculate domain and range

Reply:
It is fine if it works this way